src/app/components/header/header.component.html

```diff
@@ -1,17 +1,33 @@
 <header class="header">
-  <div class="img"><img src="../../../assets/img/sapex.png" alt=""></div>
-  <nav class="user-buttons">
-    <a routerLink="/users" routerLinkActive="active" (click)="showSpinner()" mat-raised-button *ngIf="role" id="candidatesList">{{ 'header.candidatesList' | translate }}</a>
-    <a routerLink="/calendar" routerLinkActive="active" mat-raised-button *ngIf="role" id="calendar">{{ 'header.calendar' | translate }}</a>
-    <a routerLink="/sandbox" routerLinkActive="active" mat-raised-button *ngIf="role" id="createNewSandbox">{{ 'header.createNewSandbox' | translate }}</a>
-  </nav>
-  <div class="username" *ngIf="userName" class="userName">{{'header.hello' | translate}}, {{ userName }}</div>
-  <div class="button-login">
-    <button mat-raised-button (click)="openModal()" *ngIf="!userName" > {{ 'login' | translate }}</button>
+  <div class="img">
+    <a routerLink="/"><img src="../../../assets/img/sapex.png" alt=""></a>
   </div>
-  <div class="button-logout" *ngIf="userName" (click)="logOut()" >
-    <button mat-raised-button >{{ 'logout' | translate }}</button>
+  <nav class="user_tabs">
+    <a routerLink="/users" routerLinkActive="active" (click)="showSpinner()" mat-button *ngIf="userName"
+      id="candidatesList">{{ 'header.candidatesList' | translate }}</a>
+    <a routerLink="/calendar" routerLinkActive="active" mat-button *ngIf="userName" id="calendar">{{
+      'header.calendar' | translate }}</a>
+    <a routerLink="/sandbox" routerLinkActive="active" mat-button *ngIf="userName" id="createNewSandbox">{{
+      'header.createNewSandbox' | translate }}</a>
+  </nav>
+  <!-- <nav class="user-buttons">
+    <a routerLink="/users" routerLinkActive="active" (click)="showSpinner()" mat-raised-button *ngIf="userName"
+      id="candidatesList">{{ 'header.candidatesList' | translate }}</a>
+    <a routerLink="/calendar" routerLinkActive="active" mat-raised-button *ngIf="userName" id="calendar">{{
+      'header.calendar' | translate }}</a>
+    <a routerLink="/sandbox" routerLinkActive="active" mat-raised-button *ngIf="userName" id="createNewSandbox">{{
+      'header.createNewSandbox' | translate }}</a>
+  </nav> -->
```

Comment:
Why is this one commented out?

Reply:
this is for choose one of styles for now